apps/sms/js/error_dialog.js

```diff
@@ -36,15 +36,33 @@
       };
     }
 
-    var messageBodyParams = null;
+    var bodyL10nId = prefix + 'Body';
+    var body = null;
+
     if (errorDescription.showRecipient) {
-      messageBodyParams = {
-        n: options.recipients.length,
-        numbers: options.recipients.join('<br />')
-      };
-    }
+      var fragment = document.createDocumentFragment();
+      var mainText = document.createElement('span');
+      navigator.mozL10n.setAttributes(
+        mainText, bodyL10nId,
+        { n: options.recipients.length }
+      );
+      fragment.appendChild(mainText);
+
+      var list = options.recipients.reduce((list, recipient) => {
+        var li = document.createElement('li');
+        li.setAttribute('dir', 'ltr'); // phone numbers are always LTR
```

Comment:
maybe we can move this to css since you already need a rule for li in FDN error dialog?